src/sessionStore/shellOptions.ts

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
Could we maybe incorporate descriptions in here as well?

Reply:
they can be added to the set command yeah